test/credentials_test.dart

```diff
@@ -299,4 +299,185 @@ Future<void> main([List<String>? args]) async {
       await authProvider.callResetPasswordFunction(username, newPassword);
     }, throws<RealmException>("failed to reset password for user $username"));
   }, appName: AppNames.autoConfirm);
+
+  /// JWT Header:
+  /// {
+  ///   "kid": "1",
+  ///   "alg": "RS256",
+  ///   "typ": "JWT"
+  /// }
+  /// JWT Payload data
+  /// {
+  ///    "sub": "62f394e9bcb9fee0c9aecb76",  //User.identities.id: If it is a new id the user is created, if it is an existing id the user and profile are updated.
+  ///    "name": {
+  ///      "firstName": "John",
+  ///      "lastName": "Doe"
+  ///    },
+  ///    "email": "JWT_privatekey_validated_user@realm.io",
+  ///    "gender": "male",
+  ///    "birthDay": "1999-10-11",
+  ///    "minAge": "10",
+  ///    "maxAge": "90",
+  ///    "company": "Realm",
+  ///    "iat": 1660130537,
+  ///    "exp": 1660349537, //10 years after Aug 2022
```

Comment:
I typically make these expirations 100+ years as many projects may last longer than 10 years.

Reply:
I hope so. OK, I will change it.

Reply:
Done.